homeassistant/components/synology_dsm/sensor.py

```diff
@@ -105,3 +104,24 @@ def state(self):
             return display_temp(self.hass, attr, TEMP_CELSIUS, PRECISION_TENTHS)
 
         return attr
+
+
+class SynoDSMInfoSensor(SynologyDSMEntity):
+    """Representation a Synology information sensor."""
+
+    @property
+    def state(self):
+        """Return the state."""
+        attr = getattr(self._api.information, self.entity_type)
+        if attr is None:
+            return None
+
+        # Temperature
+        if self.entity_type in TEMP_SENSORS_KEYS:
+            return display_temp(self.hass, attr, TEMP_CELSIUS, PRECISION_TENTHS)
+
+        if "uptime" == self.entity_type:
+            uptime = utcnow() - timedelta(seconds=attr)
+            return as_local(uptime.replace(microsecond=0)).isoformat()
```

Comment:
The timestamp should be utc time, not local time.

Reply:
Actually, I just based myself on https://github.com/home-assistant/core/blob/dev/homeassistant/components/systemmonitor/sensor.py#L225, which is using `as_local`.  I'm gonna move to utc time